### DISCUSSION TITLE
DUDA

### DISCUSSION THREAD

**Discussion by CRIS-117** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#567] [Q&A]
Por cuestiones laborales y de escuela no e podido entrar a los lives, ya no va haber nada para esta semana?

**Comment by Ingrzs** [comment]
Mañana jueves habrá live alas 7:00 pm nos darán la información para la misión backend.
El live será en Instagram o en Team

**Comment by JuanSalvador73** [comment]
Hubo un live el lúnes para manejo de streés, básicamente te enseñan a "meditar". El otro será este jueves a las 7:00 por Teams. Ese ya nos indicarán como proceder para la siguiente etapa.

**Comment by doguedogue** [comment]
será hasta el viernes 1ABR22
